react/src/Datasets/components/DatasetInfoDialog.tsx

```diff
@@ -68,15 +67,6 @@ export default function DatasetInfoDialog({ dataset_id, onClose, open }: DialogP
         const newData = fields
             .map(field => {
                 if (field.fieldName && !field.disableEdit) {
-                    if (
```

Comment:
This section of code is responsible for reformatting the value in `Library Prep Date` as a string that matches a certain regex such that the backend can interpret correctly the request as a Date object. Without this reformatting step, the backend would just interpret the request as a string and is unable to convert it to a Date object. 

Right now, since this section is deleted and not added anywhere else, we would observe a 500 INTERNAL SERVER error when changes to `Library Prep Date` are made.